src/quicr_server.cpp

```diff
@@ -226,15 +192,17 @@ QuicRServer::handle_publish(
 {
   messages::PublishDatagram datagram;
   msg >> datagram;
+
   // TODO: Add publish_state when we support PublishIntent
 
-  delegate.onPublisherObject(datagram.header.name,
-                             context_id,
-                             mStreamId,
-                             0,
-                             0,
-                             false,
-                             std::move(datagram.media_data));
+  //  std::ostringstream log_msg;
+  //  log_msg << "handle_publish: cid: " << context_id << " sid: " << mStreamId
+  //          << " gid: " << from_varint(datagram.header.group_id)
+  //          << " oid: " << from_varint(datagram.header.object_id)
+  //          << " off: " << from_varint(datagram.header.offset_and_fin);
+  //  log_handler.log(qtransport::LogLevel::info, log_msg.str());
```

Comment:
We probably shouldn't push dead code. If we want it back we can get it from version control.

Reply:
It was there in case we need to add it back for debugging, but IMO I don't need it anymore, so I'll remove.